### PR TITLE
Update helm chart for `furiosa-metrics-exporter`

### PR DESCRIPTION
**Pull request by hoony9x-furiosa-ai**
### One line PR Description
- resolve: furiosa-ai/furiosa-metrics-exporter#27

### What type of PR is this?
/kind devops

### Special notes for reviewer
- Related PR: furiosa-ai/furiosa-metrics-exporter#28

### CODE REVIEW

File: charts/furiosa-metrics-exporter/values.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ daemonSet:
                 values:
                   - "true"
   image:
-    repository: docker.io/furiosaai/furiosa-metrics-exporter
+    repository: registry.corp.furiosa.ai/furiosa/furiosa-metrics-exporter
```
Comment (bg-furiosa) [inline charts/furiosa-metrics-exporter/values.yaml:25]: image should not be changed. public helm chart should reference public docker image.

File: charts/furiosa-metrics-exporter/templates/daemonset.yaml [hunk 2]
```diff
@@ -21,13 +21,21 @@ spec:
...
       containers:
         - name: {{ include "furiosa-metrics-exporter.name" . }}
-          image: {{ .Values.daemonSet.image.repository | default "docker.io/furiosaai/furiosa-metrics-exporter" }}:{{ .Values.daemonSet.image.tag | default "latest" }}
-          command: ["./main"]
+          image: {{ .Values.daemonSet.image.repository | default "registry.corp.furiosa.ai/furiosa/furiosa-metrics-exporter" }}:{{ .Values.daemonSet.image.tag | default "latest" }}
+          command:
+            - ./main
+            - --port=$(PORT)
+            - --interval=$(INTERVAL)
+            - --node-name=$(NODE_NAME)
```
Comment (bg-furiosa) [inline charts/furiosa-metrics-exporter/templates/daemonset.yaml:29]: why don't you use `args:`.

File: charts/furiosa-metrics-exporter/values.yaml (hunk 1 above)
Comment (hoony9x-furiosa-ai) [reply charts/furiosa-metrics-exporter/values.yaml]: Reverted at da5f0a41

File: charts/furiosa-metrics-exporter/templates/daemonset.yaml (hunk 2 above)
Comment (hoony9x-furiosa-ai) [reply charts/furiosa-metrics-exporter/templates/daemonset.yaml:28]: Fixed at 58918cac

File: charts/furiosa-metrics-exporter/Chart.yaml [hunk 3]
```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 name: furiosa-metrics-exporter
 description: A Helm chart for furiosa-metrics-exporter
 type: application
-version: 0.1.2
-appVersion: "0.1.0"
+version: 0.1.3
+appVersion: "v0.0.12"
```
Comment (bg-furiosa) [inline charts/furiosa-metrics-exporter/Chart.yaml:6]: any reason to update appVersion?

File: charts/furiosa-metrics-exporter/templates/daemonset.yaml [hunk 4]
```diff
@@ -21,13 +21,20 @@ spec:
       priorityClassName:  {{ .Values.daemonSet.priorityClassName | default "system-node-critical" }}
       containers:
         - name: {{ include "furiosa-metrics-exporter.name" . }}
-          image: {{ .Values.daemonSet.image.repository | default "docker.io/furiosaai/furiosa-metrics-exporter" }}:{{ .Values.daemonSet.image.tag | default "latest" }}
-          command: ["./main"]
+          image: {{ .Values.daemonSet.image.repository | default "registry.corp.furiosa.ai/furiosa/furiosa-metrics-exporter" }}:{{ .Values.daemonSet.image.tag | default "latest" }}
```
Comment (bg-furiosa) [inline charts/furiosa-metrics-exporter/templates/daemonset.yaml:24]: don't try to fix defaulting :)

File: charts/furiosa-metrics-exporter/Chart.yaml (hunk 3 above)
Comment (hoony9x-furiosa-ai) [reply charts/furiosa-metrics-exporter/Chart.yaml:6]: Just want to match with our [latest release](https://github.com/furiosa-ai/furiosa-metrics-exporter/releases/tag/v0.0.12).

File: charts/furiosa-metrics-exporter/templates/daemonset.yaml (hunk 4 above)
Comment (hoony9x-furiosa-ai) [reply charts/furiosa-metrics-exporter/templates/daemonset.yaml]: Oh, It just copied from [this repo](https://github.com/furiosa-ai/furiosa-metrics-exporter) by mistake.

Fixed at 45932130

File: charts/furiosa-metrics-exporter/Chart.yaml (hunk 3 above)
Comment (bg-furiosa) [reply charts/furiosa-metrics-exporter/Chart.yaml:6]: for now we use "0.1.0" for all helm charts

**Review by bg-furiosa — APPROVED** [review]
conditional approval:
- fix appVersion to "0.1.0"

**Comment by hoony9x-furiosa-ai** [comment]
> conditional approval:
> 
> * fix appVersion to "0.1.0"

`appVersion` rollbacked to `0.1.0`.

**Pull request merged**